src/libs/migrations/CheckForPreviousReportActionID.ts

```diff
@@ -1,17 +1,14 @@
-import _ from 'underscore';
-import Onyx from 'react-native-onyx';
-import Log from '../Log';
+import Onyx, {OnyxCollection} from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
+import * as OnyxTypes from '../../types/onyx';
+import Log from '../Log';
 
-/**
- * @returns {Promise<Object>}
- */
-function getReportActionsFromOnyx() {
+function getReportActionsFromOnyx(): Promise<OnyxCollection<OnyxTypes.ReportActions>> {
     return new Promise((resolve) => {
         const connectionID = Onyx.connect({
             key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
             waitForCollectionCallback: true,
-            callback: (allReportActions) => {
+            callback: (allReportActions: OnyxCollection<OnyxTypes.ReportActions>) => {
```

Comment:
No need for the type:
```suggestion
            callback: (allReportActions) => {
```